test/uk/gov/hmrc/securemessage/controllers/SecureMessageControllerSpec.scala

```diff
@@ -53,29 +55,84 @@ class SecureMessageControllerSpec extends PlaySpec with ScalaFutures with Mockit
 
   "createConversation" must {
 
-    "return CREATED (201) when sent a request with all optional fields populated" in new CreateConversationTestCase(
+    s"return CREATED ($CREATED) when sent a request with all optional fields populated" in new CreateConversationTestCase(
```

Comment:
you can't right client and run a single test if you have string interpolations here, so better to revert this to what was before